python/django_bolt/management/commands/runbolt.py

```
@@ -1351,6 +1355,16 @@ def merge_apis(self, apis):
                 route_map[asgi_key] = api_path
                 merged._asgi_mounts.append((asgi_prefix, asgi_app))
 
+            # Merge MCP mounts from this API
+            for mcp_mount in getattr(api, "_mcp_mounts", []):
+                mcp_key = f"MCP {mcp_mount['path']}"
+                if mcp_key in route_map:
+                    raise CommandError(
+                        f"MCP mount conflict: {mcp_mount['path']} defined in both {route_map[mcp_key]} and {api_path}"
+                    )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)